news/2017-10-08-community-mentorship-program-winter-2017-signups-now-open.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "Community Mentorship Program Winter 2017 Signups Now Open"
+date: 2017-10-08 12:00:00 +0000
+---
+Have you ever opened up the editor and just looked at everything it offers in confusion? Do you have no idea where to even begin understanding it? Worry no further, the Community Mentorship Program might be the place for you!
+
+![](/wiki/shared/news/banners/community-mentorship-program-winter-2017-signups-now-open.png)
+
+Since its inception in July of 2016, the program has gone through five cycles where experienced mappers teach aspiring new mappers everything there is to know about beatmapping, from the fundamentals of handling the editor to abstract and complex concepts such as ways of structuring maps or level design!
+
+**This time around we are scouting for mentors and mentees for the gamemodes osu!, osu!catch and osu!taiko!** This iteration of the program will be hosting three out of the four gamemodes of osu! - if you're interested here's a short timeline on this:
+- Mentor signups are open from 8th October until the 15th October.
+- Mentee signups will open on the 22nd October and close on the 1st of November.
+- The 2017 Winter cycle will start on the 5th of November and end on the 28th of January 2018.
+
+During this time, the internal organization team will host various events on the server, such as assorted types of mapping contests, mapping discussions ([we also publish them on the forums as guides](https://osu.ppy.sh/community/forums/topics/514122)), or even just things like multiplayer lobbies with integrated feedback sessions!
+
+If all this sparked your interest, head over to the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/649053) to find out the program's rules as well as the signup forms!
+
+We are looking forward to seeing this batch of mentors and mentees learn mapping from each other together with the community as well as your applications!
```

Comment:
We're looking forward to seeing a brand new set of mentors and mentees figuring mapping out together!